openapi/src/parts/controlled_gcp_gcs_bucket.yaml

```diff
@@ -192,6 +192,8 @@ components:
           $ref: '#/components/schemas/GcpGcsBucketDefaultStorageClass'
         lifecycle:
           $ref: '#/components/schemas/GcpGcsBucketLifecycle'
+        cloningInstructions:
```

Comment:
style q: the other metadata fields `name` and `description` are properties of the request object (`UpdateControlledGcpGcsBucketRequestBody`), so one level up from `GcpGcsBucketUpdateParameters`, while `defaultStorageClass` and `lifecycle` are GCS-specfic things we change in GCP. Should cloningInstructions be a property alongside the other resource metadata, or are these split for a different reason?

Reply:
Good question. I figured it was a less intrusive change to add it to the update parameters, where other new updatable things would likely go. Didn't consider the GCP angle.